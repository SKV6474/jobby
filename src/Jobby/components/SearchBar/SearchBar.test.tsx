import renderer from "react-test-renderer";
import Searchinput from ".";

function handleSearching(input: string): void {}

it("renders correctly", () => {
  const tree = renderer
    .create(<Searchinput onChange={handleSearching} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
