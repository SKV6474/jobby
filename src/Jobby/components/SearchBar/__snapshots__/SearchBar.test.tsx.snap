// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-beqWaB gbBvTZ"
>
  <input
    className="sc-gueYoa hJIGrp"
    id="Value"
    onKeyUp={[Function]}
    placeholder="Search"
  />
  <i
    className="sc-dmqHEX gLYtcS fa-solid fa-magnifying-glass"
    onClick={[Function]}
  />
</div>
`;
